source/game_sa/Scripts/Commands/PlayerCommands.cpp

```diff
@@ -113,166 +118,489 @@ bool IsPlayerTouchingObjectOnFoot(CPlayerPed& player, CObject& object) {
+    /* 0210 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_TURN_PLAYER_TO_FACE_CHAR);
+    /* 0222 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_SET_PLAYER_HEALTH);
+    /* 0225 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_GET_PLAYER_HEALTH);
+    /* 0230 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_STOP_PLAYER_LOOKING);
+    /* 0234 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_SET_GANG_PLAYER_ATTITUDE);
+    /* 023A */ REGISTER_UNSUPPORTED_COMMAND_HANDLER(COMMAND_IS_PLAYER_TOUCHING_OBJECT_ON_FOOT, IsPlayerTouchingObjectOnFoot);
+    /* 0246 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_SET_ANIM_GROUP_FOR_PLAYER);
+    /* 0292 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_SET_PLAYER_HEED_THREATS);
+    /* 029F */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_IS_PLAYER_STOPPED);
+    /* 02AD */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_IS_PLAYER_IN_ANGLED_AREA_2D);
+    /* 02AE */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_IS_PLAYER_IN_ANGLED_AREA_ON_FOOT_2D);
+    /* 02AF */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_IS_PLAYER_IN_ANGLED_AREA_IN_CAR_2D);
+    /* 02B0 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_IS_PLAYER_STOPPED_IN_ANGLED_AREA_2D);
+    /* 02B1 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_IS_PLAYER_STOPPED_IN_ANGLED_AREA_ON_FOOT_2D);
+    /* 02B2 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_IS_PLAYER_STOPPED_IN_ANGLED_AREA_IN_CAR_2D);
+    /* 02B3 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_IS_PLAYER_IN_ANGLED_AREA_3D);
+    /* 02B4 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_IS_PLAYER_IN_ANGLED_AREA_ON_FOOT_3D);
+    /* 02B5 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_IS_PLAYER_IN_ANGLED_AREA_IN_CAR_3D);
+    /* 02B6 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_IS_PLAYER_STOPPED_IN_ANGLED_AREA_3D);
+    /* 02B7 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_IS_PLAYER_STOPPED_IN_ANGLED_AREA_ON_FOOT_3D);
+    /* 02B8 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_IS_PLAYER_STOPPED_IN_ANGLED_AREA_IN_CAR_3D);
+    /* 02D5 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_IS_PLAYER_SHOOTING_IN_AREA);
+    /* 02D7 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_IS_CURRENT_PLAYER_WEAPON);
+    /* 02DE */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_IS_PLAYER_IN_TAXI);
+    /* 02DF */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_IS_PLAYER_SHOOTING);
+    /* 0322 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_EXPLODE_PLAYER_HEAD);
+    /* 0336 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_SET_PLAYER_VISIBLE);
+    /* 035E */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_ADD_ARMOUR_TO_PLAYER);
+    /* 0367 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_START_KILL_FRENZY_HEADSHOT);
+    /* 0368 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_ACTIVATE_MILITARY_CRANE);
+    /* 0369 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_WARP_PLAYER_INTO_CAR);
+    /* 037F */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_GIVE_PLAYER_DETONATOR);
+    /* 03C1 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_STORE_CAR_PLAYER_IS_IN_NO_SAVE);
+    /* 03F6 */ REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_MAKE_PLAYER_UNSAFE);
```

Comment:
i can't figure it out, are these comments before or after the macro?
if before, please put them to be after.

Reply:
sorry, what macro?

Reply:
`REGISTER_COMMAND_UNIMPLEMENTED`, eg.:
```suggestion
    REGISTER_COMMAND_UNIMPLEMENTED(COMMAND_MAKE_PLAYER_UNSAFE); /* 03F6 */ 
```

Reply:
I added it before so they can be sorted by ID (as they normally go in any opcode database). They are also uniformly aligned (comment is the same length for all commands). Adding after will lose both benefits, so I'd rather remove them to reduce the noise. What do you think?

Reply:
Yeah that would work too.
The command ID can be obtained from the enum as well, so no need to have that comment there as well.

Reply:
Are you ok with adding some comment for command handler to include opcode id? e.g.
```
/// 01F5 GET_PLAYER_CHAR
int32 GetPlayerChar(CPlayerPed& player) {
    return GetPedPool()->GetRef(&player);
}
```

I feel like there should be way to quickly find command implementation knowing only its ID (using enum requires a few extra hoops)

Reply:
`/// GET_PLAYER_CHAR(01F5)`?

Reply:
I guess it works too. I tried to match format of https://library.sannybuilder.com/#/sa/script/extensions/default/01F5